jans-cedarling/cedarling/src/authz/entities/user.rs

```diff
@@ -0,0 +1,321 @@
+/*
+ * This software is available under the Apache-2.0 license.
+ * See https://www.apache.org/licenses/LICENSE-2.0.txt for full text.
+ *
+ * Copyright (c) 2024, Gluu, Inc.
+ */
+
+use super::{CreateCedarEntityError, DecodedTokens, EntityMetadata, EntityParsedTypeName};
+use crate::common::cedar_schema::CedarSchemaJson;
+use crate::common::policy_store::{PolicyStore, TokenKind};
+use cedar_policy::EntityUid;
+use std::collections::HashSet;
+use std::fmt;
+
+/// Create user entity
+pub fn create_user_entity(
```

Comment:
In this function, for creating `user` entity,
I believe it will be better at first try to map from userinfo token (it show extended info about user).
Eventually Mike want to merge id_token info and userinfo_token and from result dict create `User` entity.
And chat GPT said that info from `userinfo_token` have bigger priority about user info than `id_token`

And I'm not sure if `access_token` show information about user. So probably`access_token` should be at the end of queue..

Reply:
sure, i switched the order here: [ddd9c27](https://github.com/JanssenProject/jans/pull/10436/commits/ddd9c27e9388533ea25e50557c883bdaa9f3752b).

creating the user entity using the `access_token` is also removed for this PR and will be implemented in issue #10466 instead.